src/lib/utils/find-up.ts

```diff
@@ -0,0 +1,31 @@
+import { existsSync } from 'fs';
```

Comment:
This can be removed entirely version command is updated to require the JSON manifest
```ts
import { VERSION as ROLLUP_VERSION } from 'rollup';
import { VERSION as COMPILER_VERSION } from '@angular/compiler';
import { version as TS_VERSION } from 'typescript';
import { Command } from './command';

/**
 * Prints version information.
 *
 * @stable
 */
export const version: Command<any, void> = (pkg: any) => {
  let TSICKLE_VERSION = '';
  try {
    TSICKLE_VERSION = require('tsickle/package.json').version;
  } catch {}

  console.log(`ng-packagr:            ` + require('../package.json').version);
  console.log(`@angular/compiler:     ` + COMPILER_VERSION.full);
  console.log(`rollup:                ` + ROLLUP_VERSION);
  console.log(`tsickle:               ` + TSICKLE_VERSION);
  console.log(`typescript:            ` + TS_VERSION);
};

```

Reply:
The problem is that the path is relative to the current module (`.../lib/commands`) and since the `package.json` is copied into the destination in `postinstall` and the relative location changes.

Instead of using a relative path directly, we should pass it as an argument and also have a short comment, like:
```ts
  // NOTE: This path is correct only after postinstall.
  const packageJsonPath = path.resolve(module.path, '../package.json');
  version(packageJsonPath);
```

Reply:
That is the reason why I used `require` and not a `import`.

Require will not be analysed by the compiler, and hence it will not cause the build to fail because the package manifest path doesn’t exist doesn’t build time.

Reply:
Done.